tasks/debian.yml

```diff
@@ -12,7 +12,7 @@
     name:
       - "apt-transport-https"
       - "software-properties-common"
-      - "python-mysqldb"
+      - "python3-pymysql"
```

Comment:
Thank you for your feedback. Seeing the PR again I realise it could use some work, in particular, breaking it down in smaller ones. Please give me a couple of days to follow it up.

Reply:
Sounds good to me. And your work is much appreciated.